dhis-2/dhis-services/dhis-service-administration/src/main/java/org/hisp/dhis/dataintegrity/DefaultDataIntegrityService.java

```diff
@@ -963,6 +963,18 @@ private Set<String> expandChecks( Set<String> names )
         return expanded;
     }
 
+    private Set<String> getDefaultChecks()
+    {
+        ensureConfigurationsAreLoaded();
+
+        return checksByName.entrySet()
```

Comment:
Since you never need the key of the map you should directly stream the `values()`. This will also allow to `filter(not(DataIntegrityCheck::isSlow))` and `map(DataIntegrityCheck::getName)` like that. (`not` is on `Predicate`)

Reply:
Thanks for the tip. Fixed now.